docs/layouts/shortcodes/build_table.html

```diff
@@ -0,0 +1,35 @@
+<style type="text/css">
+  .build-table  {border-collapse:collapse;border-spacing:0;}
+  .build-table th{font-weight: bold;}
+  .build-table th span.reduce-impact{font-weight: normal;}
+  .build-table span.code{color:#905;background-color:#DDD}
```

Comment:
```suggestion
```

If we want to change the style for `<code>` elements we should do it at the theme level rather than just for this table - it might also make sense to do in a separate PR.

Reply:
done ! great idea... cleaning up this to rely on the default styling makes a lot more sense!